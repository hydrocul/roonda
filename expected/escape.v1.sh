#!/bin/sh

#################################################
# roonda_c1ee635619385ad9a559f9b6a845d10f6fdaf559.sh:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_c1ee635619385ad9a559f9b6a845d10f6fdaf559.sh
echo -n 'Hello, world!
'
echo -n 'Say "Hello, world!"
'
echo -n '\\
'
echo -n '\\nx
'
END_OF_ROONDA_SOURCE_FILE
#################################################

sh $ROONDA_TMP_PATH/roonda_c1ee635619385ad9a559f9b6a845d10f6fdaf559.sh
