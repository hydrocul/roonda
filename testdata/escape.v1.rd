sh v1

# v2にも同様のテストあり

(sh
 (print "Hello, world!\n")
 (print "Say \"Hello, world!\"\n")
 (print "\\\n")
 (print "\\nx\n"))

