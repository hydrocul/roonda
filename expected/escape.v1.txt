Hello, world!
Say "Hello, world!"
\
